### ISSUE TITLE
Trying to count files...

### ISSUE THREAD

**Issue by iopss** [ilastik/ilastik#1201]
Hi,

This might be the wrong place to post this... but the ilastik user mailing list seems to be down.

So, am a clumsy new user with Ilastik... and it might as well be an overkill to the problem I have.
I need to count flies in some images. The quality of the images was not good and varied a lot, so I have marked many files of flies myself. So, now I have several "binary" black over white files.
The images get very easily processed with Ilastik, and the only thing I want to do is count the dots.

I have gotten as far as doing an object classification file and having two classes of labels (fruit flies and big flies), an chosen from the subset features, only from the standard object feature, count.
**So, my first doubt:** Is this what I think it is? Will this just count the objects classified in the image?

When I get to the export settings, the program wants to export an image with the feature table export. I just need to export the information of the counting of objects.
**Second doubt:** Is it possible to only generate a cvs file with the name of the image file, followed by the count of objects?

I would also need to do this with some 200 images
**Third doubt:** Would it be possible to generate a single output file with the counted number of all images?

**Comment by akreshuk** [comment]
> Is this what I think it is? Will this just count the objects classified in the image?
No, this feature has  a misleading name. It's the size of the object ('pixel count').

>  Is it possible to only generate a cvs file with the name of the image file, followed by the count of objects?
Perhaps someone will correct me, but I don't think we return the count of objects. You can, however, save a csv file with label predictions for each object and count how many were assigned to each class. 

> Would it be possible to generate a single output file with the counted number of all images?
You can get ilastik to generate a single output file, but it would still save per-object information. To generate one file, load your images as a stack and below the stack menu you'll see two options: stack across t and stack across z. If you stack across t, the images will be handled one by one, but a single output file will be generated.

In general, displaying/exporting the object count for each class is not difficult and people ask for it quite frequently. I'll create another issue and link yours there.

**Comment by akreshuk** [comment]
That would be issue #1139.
